### ISSUE TITLE
kraken.config is iterating 5 times 

### ISSUE THREAD

**Issue by DStorck** [samsung-cnct/k2#416]
even with only one cluster, this task is being looped over five times: 
``` TASK [kraken.config : include] *************************************************
included: /kraken/ansible/roles/kraken.config/tasks/setup.yaml for localhost
included: /kraken/ansible/roles/kraken.config/tasks/setup.yaml for localhost
included: /kraken/ansible/roles/kraken.config/tasks/setup.yaml for localhost
included: /kraken/ansible/roles/kraken.config/tasks/setup.yaml for localhost
included: /kraken/ansible/roles/kraken.config/tasks/setup.yaml for localhost
```

**Comment by coffeepac** [comment]
So we are looping per node pool id guess.   Is that something we need to do?

**Comment by DStorck** [comment]
I think this will be fixed by the removal of the non-used loop vars fixed by https://github.com/samsung-cnct/k2/pull/419 . I had just seen lots of repetition when I was checkin the CI jobs ticket and it seemed like funny business

**Comment by DStorck** [comment]
This has been fixed by the removal of the node loop in #419
